app/javascript/src/Policies/components/HeaderButton.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import * as React from 'react'
+
+import type { ThunkAction } from 'Policies/types'
+
+type Props = {
+  type: 'add' | 'cancel',
+  onClick: () => ThunkAction,
+  children?: React.Node
+}
+
+const getIcon = type => {
+  switch (type) {
```

Comment:
is there anything in lodash like Ramda's cond? https://ramdajs.com/docs/#cond ?

Reply:
Yes there is, the same function in fact.
https://www.npmjs.com/package/lodash.cond

Reply:
It doesn't look very good IMO:
```javascript
const getIcon = type => [
  [ type === 'add', 'fa fa-plus-circle' ],
  [ type === 'cancel', 'fa fa-times-circle' ]
]
```
I changed for a dictionary, I think it's looks simpler now.

Reply:
yeah, much better! 🏅